region/brinstar/pink/Pink Brinstar Hopper Room.json

```diff
@@ -812,6 +854,71 @@
       "reusableRoomwideNotable": "Pink Hoppers Shinespark into the Wall to Ride the Elevator",
       "note": "Shinesparking horizontally into the wall will trigger the elevator as Samus's echos hit it."
     },
+    {
+      "link": [2, 3],
+      "name": "Pink Hoppers X-Ray Climb Spring Ball Jump",
+      "entranceCondition": {
+        "comeInWithDoorStuckSetup": {}
+      },
+      "requires": [
+        "canXRayClimb",
+        "canTrickyJump",
+        "canLateralMidAirMorph",
+        "canTrickySpringBallJump",
+        "canSpringFling"
+      ],
+      "note": [
+        "X-Ray climb until Samus is a little over halfway off screen; the position is not precise.",
+        "Face right, then do a turnaround buffer spin jump into a Spring Ball jump to reach the ledge.",
+        "The pause input can be pressed at the same time as the turnaround.",
+        "It helps to do an airball to gain horizontal speed.",
+        "The jump input after pausing should not be buffered, in order to allow a little time for the spring fling horizontal speed to take effect."
+      ]
+    },
+    {
+      "link": [2, 3],
+      "name": "Pink Hoppers X-Ray Climb HiJump Insane Jump",
+      "entranceCondition": {
+        "comeInWithDoorStuckSetup": {}
+      },
+      "requires": [
+        "HiJump",
+        "canXRayClimb",
+        "canInsaneJump"
+      ],
+      "note": [
+        "Get stuck 3 pixels inside the door, and X-ray climb to a precise height:",
+        "after just one pixel of Samus' feet are visible while crouching facing left, perform exactly 7 more X-Ray stand-ups.",
+        "Use X-ray to face left, then run and jump to barely make it onto the ledge."
+      ],
+      "devNote": [
+        "3 pixels stuck in the door is X position 478 (0x1DE)."
```

Comment:
This information would be useful to show on the strat page but also doesn't fit as part of a normal note.

Reply:
Yeah I've thought about adding a button or something that could reveal any devNotes on a strat, or maybe the full JSON for the strat.